### ISSUE TITLE
Resnet 학습 부분에서 궁금한 부분이 있습니다.

### ISSUE THREAD

**Issue by DonghoonPark12** [taki0112/ResNet-Tensorflow#5]
안녕하세요. 조금 혼란 스러운 부분이 있어서 질문을 드립니다.
[해당 라인](https://github.com/taki0112/ResNet-Tensorflow/blob/01dfbda7f9a0898e39fdb9e4bad651369d2921e8/ResNet.py#L206)에서 한 배치마다 Test error를 계산한 것을 볼 수 있었습니다. validation 처럼 사용하신 것 같은데요. 그래서 작성하신 test_loss, test_accuracy를 전 val_loss, val_accuracy로 이해했습니다.  그런데 [데이터 전처리](https://github.com/taki0112/ResNet-Tensorflow/blob/01dfbda7f9a0898e39fdb9e4bad651369d2921e8/utils.py#L188)에서 validation 데이터를(실제로test 데이터로 이름 붙이신) Train 데이터 mean, std로 노말라이즈 한것을 볼 수 있었는데요.

제가 알기론 validation 데이터는 validation mean, std로 노말라이즈 하는 걸로 알고 있는데 제가 잘 못 알고 있는 건가요?

아니면 코드 그대로 배치마다 테스트 데이터를 돌려 보신건가요?. 궁금합니다.

**Comment by DonghoonPark12** [comment]
두번째가 맞다면 [DenseNet 학습 부분 전처리](https://github.com/taki0112/Densenet-Tensorflow/blob/5a1b4b4e82e24f10903d2165a00891025ef59683/Cifar10/Densenet_Cifar10.py#L201) 에서는 테스트 데이터 노말라이즈를 테스트 데이터 mean, std로 했는데 위의 것과 차이가 있지 않나 생각 됩니다.

**Comment by taki0112** [comment]
train데이터에서 사용한 mean, std로 normalization을 하는게 맞습니다.
densenet의 경우, 제가 당시에 고치질 못햇었네요.. -_- ;; 죄송합니다.

**Comment by DonghoonPark12** [comment]
빠른 답변 감사합니다.
